src/Event/BoltFormsEvent.php

```diff
@@ -68,12 +55,7 @@ public function setData($data): void
         if (in_array($this->formsEventName, [FormEvents::PRE_SUBMIT, BoltFormsEvents::PRE_SUBMIT], true)) {
             $this->event->setData($data);
         } else {
-            throw new \RuntimeException(self::class . '::' . __FUNCTION__ . ' can only be called in BoltFormsEvents::PRE_SUBMIT or FormEvents::PRE_SUBMIT');
+            throw new RuntimeException(self::class . '::' . __FUNCTION__ . ' can only be called in BoltFormsEvents::PRE_SUBMIT or FormEvents::PRE_SUBMIT');
         }
     }
-
-    public function getForm(): FormInterface
```

Comment:
Since I cannot comment on a line between changes (this comment is meant for line 48): perhaps remove `getData` too since it only differs in the return type w.r.t. the parent class?